apps/metastore/src/metastore/templates/metastore.mako

```diff
@@ -355,46 +355,64 @@ ${ components.menubar(is_embeddable) }
   <!-- /ko -->
 </script>
 
+
+<script type="text/html" id="metastore-table-health">
+  <!-- ko if: tableDetails() && ! tableDetails().is_view -->
+    <!-- ko with: tableHealth -->
+    <h4>${ _('Health') }</h4>
+    <div class="row-fluid">
+      <!-- ko with: $parent.tableHealth -->
+
+      <!-- ko if: status == 'unknown' -->
+      <div class="label">${_('Unknown')}</div>
+      <!-- /ko -->
+
+      <!-- ko if: status == 'healthy' -->
+      <div class="label label-success">
+        ${_('Last update was')}
+        <!-- ko if: last_update_ago.days > 0 -->
+        <span data-bind="text: last_update_ago.days"></span> day(s)
+        <!-- /ko -->
+        <span data-bind="text: last_update_ago.hours"></span> hour(s)
+        ago <br/>
+        (<span data-bind="text: localeFormat(last_update * 1000)"></span>)
+      </div>
+      <!-- /ko -->
+
+      <!-- ko if: status == 'unhealthy' -->
+      <div class="label label-warning">
+        ${_('Last update was')}
+        <!-- ko if: last_update_ago.days > 0 -->
+        <span data-bind="text: last_update_ago.days"></span> day(s)
+        <!-- /ko -->
+        <span data-bind="text: last_update_ago.hours"></span> hour(s)
+        ago <br/>
+        (<span data-bind="text: localeFormat(last_update * 1000)"></span>)
+      </div>
+      <!-- /ko -->
+
+      <!-- ko if: status !== 'unknown' -->
+      <div>
+        <i class="fa fa-fw fa-clock-o muted" ></i>
+        <strong>${_('Next in')}</strong>
+        <!-- ko if: next_update_expected_ago.days > 0 -->
+        <span data-bind="text: next_update_expected_ago.days"></span> day(s)
+        <!-- /ko -->
+        <span data-bind="text: next_update_expected_ago.hours"></span> hour(s)<br/>
+        (<span data-bind="text: localeFormat(next_update_expected * 1000)"></span>)
+      </div>
+      <!-- /ko -->
+
+      <!-- /ko -->
+    </div>
+    <!-- /ko -->
+  <!-- /ko -->
+</script>
+
+
 <script type="text/html" id="metastore-databases">
   <div class="actionbar-actions" data-bind="dockable: { scrollable: '${ MAIN_SCROLLABLE }', nicescroll: true, jumpCorrection: 5, topSnap: '${ TOP_SNAP }' }">
     <input class="input-xlarge search-query margin-left-10" type="text" placeholder="${ _('Search for a database...') }" data-bind="clearable: databaseQuery, value: databaseQuery, valueUpdate: 'afterkeydown'"/>
-    % if has_write_access:
```

Comment:
let's just wrap it into `<!-- ... -->` to now remove the code and highlight why we did certain decisions